lib/iruby/display.rb

```diff
@@ -106,14 +106,15 @@ def protect(mime, data)
       # but mime-types library tells us it is a non-text type.
       FORCE_TEXT_TYPES = Set[
         "application/javascript",
-        "image/svg+xml"
+        "image/svg+xml",
+        "text.html",
```

Comment:
MIME Types must have a `/` in the string.  Was the intention to  add `text/html` here?